### PR TITLE
1.1-DRAFT1

### PR DESCRIPTION
**Pull request by jayshao**
Starting a pull request for comments or discussion for the 1.1 DRAFT

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,17 +18,22 @@ Mapping to other taxonomies
...
+* [M. Wasfi](https://github.com/mowasfi7) (Verizon)
+* Matthew Mercuri (Samsung)
+* [Nicholas Babb](https://github.com/ndbabb) (Adomni)
+* Sweta Vasudevan (Place Exchange)
+* William Hartley-Booth (The Trade Desk)
 
 ## Adoption
 
-| Participant | Status | Version | Notes |
+| Platform | Status | Version | Notes |
```
Comment (bruno8g) [inline README.md:36]: Hivestack is now supporting the taxonomy 1.0, can we add our name to the list

File: specification-1.0.md [hunk 2]
```diff
@@ -179,142 +188,159 @@ String values are text-based identifiers of venue categories. Dependencies on pa
...
+### Retail: Fueling Stations
 
-| Grandchild Category | Category Definition | Enumeration ID | String Value               |
-| ------------------- | ------------------- | -------------- | -------------------------- |
-| Pump                | TBC                 | 20101          | retail.gas\_stations.pump  |
-| Shop                | TBC                 | 20102          | retail.gas\_stations.shop  |
+| Grandchild Category | Category Definition                                                              | Enumeration ID | String Value (Deprecated)  |
+| ------------------- | -------------------------------------------------------------------------------- | -------------- | -------------------------- |
+| Fuel Dispenser      | A (typically self-service) device for dispensing fuel to vehicles.               | 20101          | retail.gas\_stations.pump  |
+| Shop                | A store attached to a location who's primary audience if people fueling vehicles | 20102          | retail.gas\_stations.shop  |
```
Comment (bruno8g) [inline specification-1.0.md:255]: "is" instead of "if"

File: README.md (hunk 1 above)
Comment (jayshao) [reply README.md:36]: @bruno8g when did you guys start supporting it?

File: specification-1.0.md [hunk 3]
```diff
@@ -179,142 +188,159 @@ String values are text-based identifiers of venue categories. Dependencies on pa
...
 
 ### Transit: Train Stations
 
-| Grandchild Category | Category Definition | Enumeration ID | String Value                     |
-| ------------------- | ------------------- | -------------- | -------------------------------- |
-| Train               | TBC                 | 10601          | transit.train\_stations.train    |
-| Platform            | TBC                 | 10602          | transit.train\_stations.platform |
+| Grandchild Category | Category Definition                                          | Enumeration ID | String Value (Deprecated)        |
+| ------------------- | ------------------------------------------------------------ | -------------- | -------------------------------- |
+| Train               | A train that travels primarily above ground, on rails\       | 10601          | transit.train\_stations.train    |
```
[the remark this answers is not in the document]
Comment (jayshao) [reply specification-1.0.md:247]: The \ should be proper markdown to escape the "_" and make sure that it's not interpreted as emphesis - I don't see it in the rendered document

**Comment by jayshao** [comment]
Publishing 1.1-DRAFT1 for public comment

**Pull request merged**